pkg/agent/openflow/client.go

```diff
@@ -590,7 +590,7 @@ func (c *client) InstallGatewayFlows() error {
 func (c *client) InstallDefaultTunnelFlows() error {
 	flows := []binding.Flow{
 		c.tunnelClassifierFlow(config.DefaultTunOFPort, cookie.Default),
-		c.l2ForwardCalcFlow(globalVirtualMAC, config.DefaultTunOFPort, true, cookie.Default),
+		c.l2ForwardCalcFlow(globalVirtualMAC, config.DefaultTunOFPort, false, cookie.Default),
```

Comment:
This would introduce some overhead the original pipeline design wanted to avoid. I think the reply packets of ingress traffic don't need to go to ingress table, it could just go to ingress metric table. The `ct_label` is persistent in conntrack, doesn't need to be set for every packet.